lib/Team.js

```diff
@@ -0,0 +1,172 @@
+/**
+ * @file
+ * @copyright  2013 Michael Aufreiter (Development Seed) and 2016 Yahoo Inc.
```

Comment:
Personally I don't see a need for any of that if you declare it for the repo as a whole. I'll change it to my name though.